components/properties/src/props.rs

```diff
@@ -284,6 +289,29 @@ impl BidiClass {
     pub const PopDirectionalIsolate: BidiClass = BidiClass(22);
 }
 
+/// Enumerated property Bidi_Paired_Bracket_Type.
+///
+/// Bidi_Paired_Bracket_Type represents the type of a paired bracket, either
+/// opening or closing. This property is used in the implementation of parenthesis matching.
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "datagen", derive(databake::Bake))]
+#[cfg_attr(feature = "datagen", databake(path = icu_properties))]
+#[allow(clippy::exhaustive_structs)] // newtype
+#[repr(transparent)]
+#[zerovec::make_ule(BidiPairedBracketTypeULE)]
+pub struct BidiPairedBracketType(pub u8);
+
+#[allow(non_upper_case_globals)]
+impl BidiPairedBracketType {
```

Comment:
thought: hmm, so I was under the impression we already had this type when I suggested making a closed and open.

If this is a new type, perhaps this can just be a closed enum for now (and we have a single type). @sffc thoughts?

Reply:
this is being discussed below